cpp/example/MODULE.bazel.lock

```diff
@@ -0,0 +1,160 @@
+{
```

Comment:
what's this?

Reply:
it's generated by bazel

Reply:
Do we need to check in this? I don't think we check in something like this before.

Reply:
already manually reverted in the latest commit